fpdcc/ccfp-asset-dashboard
# Configure second GIS database and router, add example model and skeleton models for remainder of quercus schema

## Overview

See title.

Closes #60

### Notes

AFAICT, the currents tests are OK, but heads up that pytest-django [only has experimental support for multiple databases](https://pytest-django.readthedocs.io/en/latest/database.html#tests-requiring-multiple-databases).

## Testing Instructions

* Run the app and load the fixture data and GIS data dump, as documented in the README.
* Open up a Django shell.
* Query for a managed model (e.g., `Section`) and confirm you get hits: `Section.objects.all()`
* Query for a `NaturePreserves` and confirm you get hits: `NaturePreserves.objects.all()`
